src/tests/unit/services/F2fService.test.ts

```diff
@@ -97,6 +97,14 @@ describe("F2f Service", () => {
 		}));
 	});
 
+	it("should throw 500 if request fails during update Session data with yoti session details", async () => {
```

Comment:
do we also need a happy path test to confirm that the call to update the session table with the YotiSessionId has taken place?

Reply:
Done :)